web/templates/setting/form.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= form_for @conn, @action, [as: :settings, method: :put], fn f -> %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for svc <- @system_variable_changesets do %>
+        <tr>
+          <td> <%= svc.data.key %> </td>
+
+          <%= inputs_for f, integer_to_atom(svc.data.id), fn ff -> %>
```

Comment:
I just saw that the user can't create params. But it's still slightly problematic because it's coming from the database and since it's supposed to be a stand-alone app, we can't make sure the table doesn't grow.